apollo-runtime/src/main/java/com/apollographql/apollo3/internal/RealApolloSubscriptionCall.kt

```diff
@@ -123,7 +128,13 @@ class RealApolloSubscriptionCall<D : Subscription.Data>(
       val data = response.response.data
       if (callback != null) {
         if (data != null && delegate!!.cachePolicy != ApolloSubscriptionCall.CachePolicy.NO_CACHE) {
-          delegate!!.apolloStore.writeOperation(response.subscription, data)
+          delegate!!.apolloStore.writeOperation(
+              response.subscription,
+              data,
+              delegate!!.responseAdapterCache,
+              CacheHeaders.NONE,
+              true,
+              )
```

Comment:
nit: could you please add names for the arguments for better readability

Reply:
Sure thing !